packages/fluentui/react-northstar/test/specs/components/Toolbar/Toolbar-test.tsx

```diff
@@ -62,4 +62,24 @@ describe('Toolbar', () => {
       expect(getRenderedAttribute(italicToolbarButton, 'aria-pressed', '')).toBe('false');
     });
   });
+
+  describe('Overflow Toolbar', () => {
+    it('Should override popper properties', () => {
+      const wrapper = mountWithProvider(
+        <Toolbar
+          aria-label="Toolbar overflow menu"
+          items={[]}
+          overflow
+          overflowItem={{ title: 'More', menu: { popper: { offset: [0, 4], position: 'below' } } }}
```

Comment:
nit: you can create the `menu` object at the beginning of the test, so you don't have to repeat yourself